test/integration/extensionRequests.test.js

```diff
@@ -214,6 +228,33 @@ describe("Extension Requests", function () {
           return done();
         });
     });
+    it("Should return success response after adding the extension request (sending assignee username)", function (done) {
```

Comment:
please write tests for error cases like not found and internal server errors.

Reply:
sure